source/simulator/solver.cc

```diff
@@ -247,6 +253,7 @@ namespace aspect
     {
       return n_iterations_A_;
     }
+
```

Comment:
Thanks for this :smile: I always get confused by too many, or too few empty lines.